### PR TITLE
First version of actual collaboration service

### LINKED ISSUE #5: Implement the first version
This will be python...... (temporary)

### PR DESCRIPTION
**Pull request by katzper-michno**
What can I say. This is temporary.

Resolves #5

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Make i run under `/collaboration` and will be ok

**Comment by ghost** [comment]
I'm debugging this right now (containerized version is not working for some reason) and find out that:
* print is not visible in docker container logs (doesn't flush properly) -- use `logging` module
```python3
import loggin
logging.getLogger().setLevel(logging.INFO)
...
logging.info("message")
```
* by default there is no distinction between endpoints, so no action required to make it run under `/collaboration`

File: server.py [hunk 1]
```diff
@@ -1,26 +1,64 @@
...
 
-clients = []
+        print(f'[CLOSED] {clientId}')
+
+document = []
+clientsConnectedIds = []
+clients = {}
+cursorsPositions = {}
 
 server = WebSocketServer('', 3400, SimpleChat)
```
Comment (ghost) [inline server.py]:
```suggestion
server = WebSocketServer('0.0.0.0', 3400, SimpleChat)
```

**Review by ghost — CHANGES_REQUESTED** [review]
Change `print` to `logging.info` as in comment above and ready to merge

File: server.py (hunk 1 above)
Comment (katzper-michno) [reply server.py]: done

**Review by ghost — APPROVED** [review]

**Pull request merged**